image/system_a/data/lang/Deutsch.json

```diff
@@ -717,4 +717,4 @@
     "tethering_menu_access_decline": "<text>Tethering ist eingeschaltet.<br /><br />Schalten Sie Tethering aus,<br />um auf das Men\u00fc zuzugreifen</text>",
     "tethering_phone_mode_change_prohibited": "<text>Tethering ist eingeschaltet.<br /><br />Andere Betriebsarten (Verbunden, DND,<br />Offline) werden von dieser Betriebsart \u00fcberlagert<br />und funktionieren nicht.</text>",
     "tethering_turn_off_question": "Tethering ausschalten?"
-}
```

Comment:
Did you remove it on purpose?

Reply:
@lukasmas No, apparently the verify_translations.py tool removes the newline automatically

Reply:
fixed